ansible/roles/products/hand-e/server/desktop-icons/templates/scripts/start-docker-container-nuc-command.j2

```diff
@@ -3,4 +3,4 @@
 NUC_NAME={{ nuc_username }}
 NUC_ADDRESS={{ nuc_address }}
 CONTAINER={{ container_name }}
-ssh -X $NUC_NAME@$NUC_ADDRESS "docker exec -i --user user $CONTAINER bash -c 'cat > ~/.Xauthority' < ~/.Xauthority; docker exec --user user $CONTAINER bash -c $'export ROSCONSOLE_FORMAT='"'[${severity}](${node}): [${time}] ${message}'"';export ROSCONSOLE_STDOUT_LINE_BUFFERED=1;source /home/user/projects/shadow_robot/base/devel/setup.bash;xauth list | tail -1 | awk \'{print \$1}\' | cut -d ':' -f2 > ~/.display_number;export DISPLAY=localhost:\$(<~/.display_number).0;/bin/update_hand_config.sh;{{ command_to_run }} 2> >(tee -a /home/user/.ros/log/stderr.log)'"
+ssh -X $NUC_NAME@$NUC_ADDRESS "docker exec -i --user user $CONTAINER bash -c 'cat > ~/.Xauthority' < ~/.Xauthority; docker exec --user user $CONTAINER bash -c $'export ROSCONSOLE_FORMAT=\'"'[${severity}](${node}): [${time}] ${message}'"\';export ROSCONSOLE_STDOUT_LINE_BUFFERED=1;source /home/user/projects/shadow_robot/base/devel/setup.bash;xauth list | tail -1 | awk \'{print \$1}\' | cut -d ':' -f2 > ~/.display_number;export DISPLAY=localhost:\$(<~/.display_number).0;/bin/update_hand_config.sh;{{ command_to_run }} 2> >(tee -a /home/user/.ros/log/stderr.log)'"
```

Comment:
Do you remember the reason we're redefining `ROSCONSOLE_FORMAT` here (or, why can't we add `ROSCONSOLE_STDOUT_LINE_BUFFERED` [here](https://github.com/shadow-robot/sr-build-tools/blob/5d77b310323b1c8956a8d508f2886c3bce6c9e9c/docker/utils/additional_bashrc_cmds_noetic#L73))?

Reply:
to answer my own question, it's because .bashrc is sourced here in non-interactive mode. Manually sourcing and passing the interactive flag would get rid of the need to re-export these variables but would need testing